src/main/java/frc/robot/subsystems/HandlerSubsystem.java

```diff
@@ -264,6 +264,42 @@ public void eject() {
     }
   }
 
+  public void slowEject() {
```

Comment:
Lots of duplicated code. We can avoid that by renaming the original `eject()` to a private `ejectImpl(ejectSpeedCoral)` (use `ejectSpeedCoral` in place of the constant), and then have `eject()` and `slowEject()` call it.

```java
public void eject() {
  ejectImpl(HandlerConstants.kSlowEjectSpeedCoral);
}
```